draft-ietf-tls-dtls-connection-id.md

```diff
@@ -345,6 +345,30 @@ data the following modification is made to the additional data calculation.
                       length_of_DTLSInnerPlaintext;
 ~~~
 
+# Peer Address Update Checks
+
+When a record with CID is received that has the source address of the enclosing
+UDP datagram different from the one previously recorded, the receiver MUST NOT
+update its view of the peer address with the source specified in the UDP packet
+before cryptographically validating the enclosed record(s).
+
+Immediately after reception, the recipient SHOULD initiate address validation
+using a Heartbeat {!RFC6520} based ping-pong with the new peer address.  This
+is to ensure that address validation can be cryptographically verified.  Both
+Heartbeat messages (request and response) MUST be wrapped into tls12_cid
```

Comment:
FMPOV, **MUST** for `request` is (currently) not possible, because the use of tls12_cid is bound to a none empty cid. Only the response can considered to be MUST tls12_cid, because the cause of this address-issues is bound to use such a none empty-cid for incoming messages.